aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/pipe/PdfPipe.java

```diff
@@ -152,7 +152,7 @@ public PipeRunResult doPipe(Message input, IPipeLineSession session) throws Pipe
 				String fileName = (String) session.get("fileName");
 				CisConversionResult cisConversionResult = null;
 				CisConversionService cisConversionService = new CisConversionServiceImpl(pdfOutputLocation, loader.getPathToExtractFonts());
-				cisConversionResult = cisConversionService.convertToPdf(binaryInputStream, fileName, saveSeparate ? ConversionOption.SEPARATEPDF : ConversionOption.SINGLEPDF);
+				cisConversionResult = cisConversionService.convertToPdf(binaryInputStream, fileName, saveSeparate ? ConversionOption.SEPERATEPDF : ConversionOption.SINGLEPDF);
 				XmlBuilder main = new XmlBuilder("main");
```

Comment:
Maar dan moet je ConversionOption zelf ook aanpassen / inchecken, en dan daar ook in commentaar erbij zetten dat de spelling zo moet zijn omdat de aspose library die waarde nodig heeft.

Reply:
Ah ik zie het, blijkbaar was dat de enige class die ik niet uit de originele custom aspose module heb verwijderd. 

Ik ga de vereiste wijzigingen toevoegen.